packs/journals/ancestries.json

```diff
@@ -602,7 +602,7 @@
gas are usually matriarchal, nagaji have a strong respect for women in leadership positions. Not all nagaji follow suit with matriarchal societies, but they tend to trace lineage through the mother, and almost no nagaji societies are strictly patriarchal.</p><h2>Alignment and religion</h2><p>Nagaji are most commonly neutral but otherwise don't have any strong pull toward specific alignments. The presence of a naga can sway nagaji toward that naga's ethos, but a nagaji's view on life tends to be determined by the culture to which they belong.</p><p>Nagaji overwhelmingly worship Nalinivati, their creator, to the point that any nagaji who pay homage to a different god over Nalinivati provoke strong negative reactions from their kin. While nagaji believe reverence for Ravithra, the supposed mother of nagas, to be proper and accept it more readily than other faiths, most believe Ravithra shouldn't be troubled with the requests of mortal adherents. A very small minority of nagaji in Jalmeray are drawn to the worship of the war god Diomazul; these adherents are tolerated by other nagaji, but seen as just as terrifying and dangerous as their patron god and usually avoided.</p><h2>Names</h2><p>Nagaji names vary depending on what region of the world they occupy, but they tend to possess short vowels when they have any at all. Nagaji usually name themselves or allow groups of siblings to name each other as children, a tradition that often sees nagaji eagerly shedding these names for new ones when they reach adulthood.</p><h3>Sample Names</h3><p>Adesha, Garija, Kaya, Kuwana, Onok, Paravata, Sheni, Takasha, Tasi, Ulu, Vaski, Yulbin</p><h2>Nagaji Mechanics</h2><p><strong>Rarity</strong> Rare</p><p><strong>Hit Points</strong> 10</p><p><strong>Size</strong> Medium</p><p><strong>Speed</strong> 25 feet</p><p><strong>Attribute Boosts</strong> Strength, Free</p><p><strong>Languages</strong> Common, Nagaji</p><p><strong>Additional Languages </strong>equal to your Intelligence modifier (if positive). Choose from Aklo, Amurrun, Aquan, Celestial, Draconic, Undercommon, Shadowtongue, Tengu, Vanaran, and any other languages to which you have access (such as the languages prevalent in your region)</p><p><strong>Traits</strong> Humanoid, Nagaji</p><p><strong>Senses</strong> Low-Light Vision</p><p><strong>Jaws</strong> Your mouth contains either rows of hooked, needle-sharp teeth or a pair of vicious serpent fangs. You gain a fangs unarmed attack that deals 1d6 piercing damage. Your fangs are in the brawling group and have the finesse and unarmed traits.</p><h2>Nagaji Heritages</h2><p>Nagaji physiology commonly varies between individuals. Choose one nagaji heritage at 1st level.</p><p>@UUID[Compendium.pf2e.heritages.Item.Hooded Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Sacred Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Titan Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Venomshield Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Whipfang Nagaji]</p>",
```

Comment:
```suggestion
                "content": "<p>With humanoid figures and serpentine heads, nagaji are heralds, companions, and servitors of powerful nagas. They hold a deep reverence for holy areas and spiritual truths, an aspect many others find as intimidating as a nagaji's appearance.</p><p>Nagaji are creations of the goddess Nalinivati, who was inspired by both humans and nagas. This inspiration, alongside nagaji's devotion to nagas, has led many to claim nagaji were created to be servants. However, Nalinivati created nagaji simply for the sake of creating. She envisioned a world where nagas and nagaji worked together to succeed, with nagas serving as sacred guardians, and nagaji living as mortals upon Golarion, respecting nagas for their strength and wisdom. Nagaji live up to this expectation, forming nations, temples, and villages with as many varied governments, societies, and traditions as there are scales on a serpent's back. If nagaji can be said to share any trait, it's devotion, be it to a community, a temple, a concept, or a lifestyle. Most nagaji are also drawn to the spiritual in all its expressions, even the darker aspects of philosophies and religions.</p><p>If you want a character who combines the crushing strength and the sinuous mystery of a serpent, you should play a nagaji.</p><h2>You Might...</h2><ul><li>Enjoy the intersection of spirituality and everyday life.</li><li>Have deep reserves of patience for tasks others might find boring.</li><li>Adhere to strong beliefs and traditions about your place in the universe.</li></ul><h2>Others Probably...</h2><ul><li>Assume you have no goals or interests beyond serving nagas.</li><li>Find your unblinking eyes and serpentine features intimidating.</li><li>Are awed by your connection and dedication to holy natural places.</li></ul><h2>Physical Description</h2><p>Nagaji are prone to sharp differences in appearance. Some have tails alongside legs, some have sharp claws on their hands and feet, and some are so unique from their kin that they're mistaken for lamias rather than nagaji. The most common nagaji features are a serpentine head and a humanoid body. They usually have the unblinking eyes of a snake, though other peoples often claim a nagaji's gaze is far more intense. Scales cover nagaji bodies, with patterns that resemble those of snakes or nagas. Depending on their heritage, a nagaji's neck might be as long as a swan's or as short as a human's, and many have a frill of skin or scales down their back. Nagaji tend to possess crushing strength, but how it manifests can vary; some have bulky figures, while others have the slender yet powerful musculature of a snake.</p><h2>Society</h2><p>Nagaji communities widely vary, from ancient empires to tiny fishing villages. They tend to be isolated from other peoples, more as a matter of convenience than choice; nagaji have different physical needs than most ancestries, so even when integrated into mixed communities, they tend to live with other nagaji. Everything from marriage traditions, religion, social roles, valued arts, and methods of governance differs in specific nagaji communities. Within these communities, however, nagaji often have very strict and traditional views on these topics. Multiple instances of wars between nagaji have come about due to one group of nagaji finding another's practices unacceptable. This perspective applies to relations with other peoples as well. For instance, most nagaji consider themselves rivals and enemies of garudas due to their historical legends, even if the nagaji involved have never even seen a garuda.</p><p>Nagaji originated in the Tian Xia region of Nagajor. Though they have since spread across Golarion to regions such as Vudra and Jalmeray, many of them still carry traditions from their ancestral homeland. Tian and Vudrani concepts of elements, medicine, and spirituality are highly common among nagaji, especially given their tendency of seeing themselves as highly pious beings. Nagaji mostly eschew the human concept of castes, beyond the vague view of a \"natural order\" that sees nagas as holy guides to mortal nagaji, but they can see the appeal of the concept of karma and the righteousness of fulfilling their proper role in the universe. While they have little physical need for clothes, as they stick to hot and humid environments, nagaji wear them with pride and treat them as decorative elements in the same manner as jewelry.</p><p>As nagas are usually matriarchal, nagaji have a strong respect for women in leadership positions. Not all nagaji follow suit with matriarchal societies, but they tend to trace lineage through the mother, and almost no nagaji societies are strictly patriarchal.</p><h2>Alignment and religion</h2><p>Nagaji are most commonly neutral but otherwise don't have any strong pull toward specific alignments. The presence of a naga can sway nagaji toward that naga's ethos, but a nagaji's view on life tends to be determined by the culture to which they belong.</p><p>Nagaji overwhelmingly worship Nalinivati, their creator, to the point that any nagaji who pay homage to a different god over Nalinivati provoke strong negative reactions from their kin. While nagaji believe reverence for Ravithra, the supposed mother of nagas, to be proper and accept it more readily than other faiths, most believe Ravithra shouldn't be troubled with the requests of mortal adherents. A very small minority of nagaji in Jalmeray are drawn to the worship of the war god Diomazul; these adherents are tolerated by other nagaji, but seen as just as terrifying and dangerous as their patron god and usually avoided.</p><h2>Names</h2><p>Nagaji names vary depending on what region of the world they occupy, but they tend to possess short vowels when they have any at all. Nagaji usually name themselves or allow groups of siblings to name each other as children, a tradition that often sees nagaji eagerly shedding these names for new ones when they reach adulthood.</p><h3>Sample Names</h3><p>Adesha, Garija, Kaya, Kuwana, Onok, Paravata, Sheni, Takasha, Tasi, Ulu, Vaski, Yulbin</p><h2>Nagaji Mechanics</h2><p><strong>Rarity</strong> Rare</p><p><strong>Hit Points</strong> 10</p><p><strong>Size</strong> Medium</p><p><strong>Speed</strong> 25 feet</p><p><strong>Attribute Boosts</strong> Strength, Free</p><p><strong>Languages</strong> Common, Nagaji</p><p><strong>Additional Languages</strong> equal to your Intelligence modifier (if positive). Choose from Aklo, Amurrun, Aquan, Celestial, Draconic, Undercommon, Shadowtongue, Tengu, Vanaran, and any other languages to which you have access (such as the languages prevalent in your region)</p><p><strong>Traits</strong> Humanoid, Nagaji</p><p><strong>Senses</strong> Low-Light Vision</p><p><strong>Jaws</strong> Your mouth contains either rows of hooked, needle-sharp teeth or a pair of vicious serpent fangs. You gain a fangs unarmed attack that deals 1d6 piercing damage. Your fangs are in the brawling group and have the finesse and unarmed traits.</p><h2>Nagaji Heritages</h2><p>Nagaji physiology commonly varies between individuals. Choose one nagaji heritage at 1st level.</p><p>@UUID[Compendium.pf2e.heritages.Item.Hooded Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Sacred Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Titan Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Venomshield Nagaji]</p><p>@UUID[Compendium.pf2e.heritages.Item.Whipfang Nagaji]</p>",
```